docs/architecture/adr-0023-text-array-object-acrs.md

```diff
@@ -0,0 +1,166 @@
+# 0023 - Object ACRs in text array
+
+## Context
+
+In Raster Foundry, we implemented a mechanism called "Access Control Rules" (ACRs) to determine, manage, and update a user's access to some first-class objects -- projects, scenes, datasources, shapes, workspaces, templates, and analyses. Such ACRs are stored in a table so that the access to an object is determined by a user's ownership, membership of a team or an organization or a platform, directly granted access, etc.
```

Comment:
There should be a link to the original ADR here and a note about this superseding it.

Also, I think we should update the original ADR to note that it is superseded by this one